install/install.go

```diff
@@ -12,6 +12,7 @@ import (
 	"fmt"
 	"os"
 	"path"
+	"slices"
```

Comment:
I think this package is still in experimental before Go 1.21.
Either need to update go.mod and CI workflows OR use `"golang.org/x/exp/slices"`